main.tf

```diff
@@ -86,6 +86,55 @@ resource "aws_s3_bucket" "default" {
       uri         = grant.value.uri
     }
   }
+
+  dynamic "replication_configuration" {
+    for_each = var.s3_replication_enabled ? [1] : []
+
+    content {
+      role = aws_iam_role.replication[0].arn
+
+      dynamic "rules" {
+        for_each = var.replication_rules == null ? [] : var.replication_rules
+
+        content {
+          id       = rules.value.id
+          priority = try(rules.value.priority, 0)
+          prefix   = try(rules.value.prefix, null)
+          status   = try(rules.value.status, null)
+
+          destination {
+            bucket             = var.s3_replica_bucket_arn
+            storage_class      = try(rules.value.destination.storage_class, "STANDARD")
+            replica_kms_key_id = try(rules.value.destination.replica_kms_key_id, null)
+            account_id         = try(rules.value.destination.account_id, null)
+
+            dynamic "access_control_translation" {
+              for_each = try(rules.value.destination.access_control_translation.owner, null) == null ? [] : [rules.value.destination.access_control_translation.owner]
+
+              content {
+                owner = access_control_translation.value
```

Comment:
`access_control_translation.value` is a list, should be get the first item from it?